.github/workflows/test_local_action.yml

```diff
@@ -12,8 +12,30 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4.1.1
 
-      - name: CCS Build action step
+      - name: CCS Build action step  - Release
         uses: ./
         with:
           project-path: 'test/working_ccs_project'
+          project-name: 'msp430_test_project'
           build-configuration: Release
+
+      - name: CCS Build action step - Debug
+        uses: ./
+        with:
+          project-path: 'test/working_ccs_project'
+          project-name: 'msp430_test_project'
+          build-configuration: Debug
+
+      - name: Upload release firmware artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: 'Firmware artifact - Release'
+          path: test/working_ccs_project/Release/msp430_test_project.txt
+          if-no-files-found: error
+
+      - name: Upload debug firmware artifact
+        uses: actions/upload-artifact@v4
+        with:
```

Comment:
You can maximise the compression level for the ti-txt files
https://github.com/actions/upload-artifact?tab=readme-ov-file#altering-compressions-level-speed-v-size